src/planscape/config/boundary.json

```diff
@@ -4,7 +4,7 @@
             "boundary_name": "task_force_regions",
             "display_name": "Wildfire Task Force Regions",
             "region_name": "none",
-            "filepath": "boundary/RFFC_Regions_2022/tier1_rffc_areas_jan2022.shp",
```

Comment:
will the filepath change break existing developer configurations (e.g. developers on their own systems)?

Reply:
The filepath change won't break local environments. This is a command that is used to load data into the database. Might make sense to update the documentation.

Let me check if there is anything about boundaries and I'll adjust in the same PR.